data_structures/queue/queue.c

```diff
@@ -4,41 +4,44 @@
  * \author [shubham patil](https://github.com/shubhamdp)
  */
 
+#include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <errno.h>
 
 /** Content of header file begins */
```

Comment:
```suggestion
/* Content of header file begins */
```
please review doxygen link in the pull-request description; the `/** */` is reserved for member documentation